cli/lsp/documents.rs

```diff
@@ -905,9 +937,14 @@ impl Documents {
       specifier.clone(),
       version,
       language_id,
+      // todo(dsherret): don't we want to pass in the headers from
+      // the cache for remote modules here in order to get the
+      // x-typescript-types?
+      None,
```

Comment:
Seems like a small bug where we don't take into account x-typescript-types for remote open documents?